frontstage/controllers/party_controller.py

```diff
@@ -0,0 +1,15 @@
+import requests
+
+from frontstage import app
+
+
+def get_respondent_by_email(email):
+
```

Comment:
Some (debug level) logging in here would be useful as well as error handling and logging of the response, e.g.:

```python
def get_respondent_by_email(email):
    logger.debug('Retrieving respondent by email')
    ...
    if response.status_code == 404:
        logger.debug('Failed to retrieve respondent by email')
        return

    try:
        response.raise_for_status()
    except requests.exceptions.HTTPError:
        logger.exception('Failed to retrieve respondent by email')
        raise ApiError(response)

    logger.debug('Successfully retrieved respondent by email')
    return response.json()
```

Reply:
Read my mind haha